frontend/src/renderer/pages/Chat/Chat.tsx

```diff
@@ -1,33 +1,43 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import config from '../../config.js';
-import {TextInput, Button} from '@mantine/core';
+import { TextInput, Button } from '@mantine/core';
 
 const Chat = () => {
   const [messages, setMessages] = useState([]);
-  const [input, setInput] = useState("");
+  const [input, setInput] = useState('');
 
   const sendMessage = async () => {
     const question = input;
     const response = await axios.post(`${config.REACT_APP_BACKEND_URL}/chat`, {
-      question: question
+      question: question,
     });
     const data = response.data;
     // your response: {"question":"How old are you?","answer":"I don't know."}
-    setMessages([...messages, {text: data.question, sender: 'user'}, {text: data.answer, sender: 'bot'}]);
-    setInput("");
+    setMessages([
+      ...messages,
+      { text: data.question, sender: 'user' },
+      { text: data.answer, sender: 'bot' },
+    ]);
+    setInput('');
```

Comment:
**praise:** once I have a "longer" chat session, the input and button disappear, would you fix that please? It's a bug in the current version already.